src/pages/index.js

```diff
@@ -1,21 +1,20 @@
 import React from "react"
 import { Link } from "gatsby"
-
-import Layout from "../components/layout"
-import Image from "../components/image"
-import SEO from "../components/seo"
+import Header from "../components/header";
+import Introduction from "../components/introduction";
+import About from "../components/about";
+import Projects from "../components/projects";
+import Education from "../components/education";
 
 const IndexPage = () => (
-  <Layout>
-    <SEO title="Home" />
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
-      <Image />
-    </div>
-    <Link to="/page-2/">Go to page 2</Link>
-  </Layout>
+  <div>
+    <Header />
```

Comment:
It is probably better to include your resume from this root component, and pass it down to its sub-components as a prop.
And within your sub-components, use object de-structuring to get the fields you need for the specific components.